### PR TITLE
Fixed importEditManually

### PR DESCRIPTION
**Pull request by PeterKuzyk**
Fixed importEditManually

### CODE REVIEW

**Review by guram21 — APPROVED** [review]

File: src/test/java/test/entity/EntityImportEditTest.java [hunk 1]
```diff
@@ -154,7 +153,7 @@ public void importEditManually() {
 
         userCreate(driver);
 
-        Assert.assertEquals(driver.findElement(By.xpath("//div[contains(text(),'" + STRING + "')]")).getText(), STRING);
+        Assert.assertEquals(driver.findElement(By.xpath("//a[contains(text(),'" + STRING + "')]")).getText(), STRING);
```
Comment (Stan575) [inline src/test/java/test/entity/EntityImportEditTest.java:156]: better would be to use String.format for xpath or find a better locator if possible

File: src/test/java/test/entity/EntityImportEditTest.java [hunk 2]
```diff
@@ -164,7 +163,7 @@ public void importEditManually() {
 
         userEdit(driver);
 
-        Assert.assertEquals(driver.findElement(By.xpath("//div[contains(text() , '" + STRING_EDIT + "')]"))
+        Assert.assertEquals(driver.findElement(By.xpath("//a[contains(text() , '" + STRING_EDIT + "')]"))
```
Comment (Stan575) [inline src/test/java/test/entity/EntityImportEditTest.java:166]: same as for line 156

**Review by Stan575 — APPROVED** [review]
Please add trello task to description

**Review by sergeonevdah — APPROVED** [review]

**Comment by oooleks** [comment]
Please **revert** your last commit:
Annotate ignore:
createApplicationTest(test.portal.AdminEntityTest)
instanceCreateTest(test.portal.PortalPrivateTemplateCreationAndUsageTest)
instanceDraftTest(test.portal.MarketplaceInstanceTest)
createApplicationTest(test.portal.AdminConstantsTest) 

as those have been resolved already

**Pull request closed without merging**